src/redux/space/spaceSlice.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-param-reassign */
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+import * as api from '../../api';
+
+export const getMissions = createAsyncThunk('space/getMissions', async () => {
+  try {
+    const response = await api.getMissions();
+    console.log(response.data);
+    return response.data;
+  } catch (error) {
+    console.log(error);
```

Comment:
:radio_button: There are a lot of console.logs in the codes. Please take them down.